src/platforms/apple/common/enriching-events/user-feedback.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "User Feedback"
+sidebar_order: 105
+
+description: "Learn more about the user feedback API, which gives you the possibility to collect user information when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
+---
+
+The user feedback API gives you the possibility to collect user information when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues.
+Sentry needs the `eventId` to be able to associate the user feedback to the corresponding event. To get the `eventId` you for example [`beforeSend`](/platforms/apple/guides/ios/configuration/options/#before-send) or use the return value of the methods capturing an event.
+
+```swift
+import Sentry
+
+let userFeedback = UserFeedack(eventId: /* Put in eventId */)
```

Comment:
What about we show:

```
let eventId = SentrySDK.capture(message: "this happened")
let userFeedback = UserFeedack(eventId: eventId)
```

So that the user can have an idea of where the eventId comes from?

Or show the snippet with beforeSend?

Reply:
yeah I think it'll be easier for the user to get it.
and also: https://develop.sentry.dev/sdk/philosophy/#assume-novice-developers